profiling/src/internal/profile/mod.rs

```diff
@@ -1062,7 +1080,7 @@ mod api_tests {
     }
 
     #[test]
-    fn lazy_endpoints() -> anyhow::Result<()> {
+    fn lazy_endpoints() {
```

Comment:
Why this change?

Reply:
It doesn't look correct for a test to exit early if something goes wrong, right: It should blow up?

Reply:
Returning an error causes the test runner to blow up.  But de gustibus non est disputandum.

Reply:
Ah! I didn't know that. In that case, these changes aren't relevant. Reverted them in f4dd6ae172ae645f4b729defcedfe7b4958ed846